cli/src/main/resources/help/user.adoc

```diff
@@ -30,6 +30,10 @@ SYNOPSIS
 
 *user roles deny* --roles='role1'[,'role2'...] 'principal'
```

Comment:
isn't `principal` and `username` the same thing? can we use the same name everywhere?

Reply:
No, a principal can also be a group name.